assets/shaders/viewport_render.wgsl

```diff
@@ -531,12 +519,12 @@ fn get_by_ray(ray_: Line) -> OctreeRayIntersection{
         }
         current_d = impact_or(current_bounds_ray_intersection, current_d);
 
-        let target_octant = node_stack[node_stack_i - 1].target_octant;
-        let target_child = current_node.children[target_octant];
-        let target_bounds = child_bounds_for(current_bounds, target_octant);
+        var target_octant = node_stack[node_stack_i - 1].target_octant;
+        var target_bounds = child_bounds_for(current_bounds, target_octant);
+        var target_child_key = current_node.children[target_octant];
```

Comment:
Local variable mutability changed because they are being used as "iterators" for the advance loop